data/SAbDab_Chen/transform.py

```diff
@@ -0,0 +1,153 @@
+import pandas as pd
+import yaml
+from tdc.single_pred import Develop
+
+
+def get_and_transform_data():
+    # get raw data
+    target_folder = "SAbDab_Chen"
+    target_subfolder = "SAbDab_Chen"
+    data = Develop(name=target_subfolder)
```

Comment:
do they thave train/valid/test splits? If yes, it would be great to add this as a column

Reply:
Added by Mohammed.